### PR TITLE
Update command descriptions in UG

### LINKED ISSUE #236: [PE-D] Description of duplicated property could be added
**Suggestions**

- UG could perhaps add a section on when an `add property` will fail. So state that two property cannot have the same address for users to watch out for such cases

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: frisciliasultan/ped#2

### LINKED ISSUE #255: [PE-D] Invalid Postal Code Acceptance Bug 2
Postal codes with more than 6 digits are accepted. As can be seen the postal code for property entry 9 has been edited to 1123456.

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: adidoesnt/ped#4

### LINKED ISSUE #257: [PE-D] Invalid Phone Number Acceptance Bug
The client contact field accepts phone numbers that are invalid either due to the digit that they start with, or because they accept phone numbers that are longer/shorter than 8 digits.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: adidoesnt/ped#6

### LINKED ISSUE #258: [PE-D] Address-PostalCode Bug
Same block can be given two different postal codes. See properties 7 and 9.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: adidoesnt/ped#7

### LINKED ISSUE #259: [PE-D] Address-Type Bug
Same building can be given two different types. See properties 7 and 9.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: adidoesnt/ped#8

### LINKED ISSUE #264: [PE-D] Date Format Documentation Bug
Documentation does not show the standard format for date input. This potentially creates functionality issues, as Feb 1, 2021 is listed after Feb 1, 20 when using sort ascending

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: adidoesnt/ped#13

### PR DESCRIPTION
**Pull request by w2vgd**
Fixes #236, Fixes #255, Fixes #257, Fixes #258, Fixes #259, Fixes #264

Is this what we want for example screenshots for commands in UG?
image

image

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditAppointmentCommand.java [hunk 1]
```diff
@@ -39,7 +39,8 @@
             + PREFIX_TIME + "1500";
 
     public static final String MESSAGE_SUCCESS = "Edited appointment: %1$s";
-    public static final String MESSAGE_DUPLICATE_APPOINTMENT = "This appointment already exists in the app";
+    public static final String MESSAGE_DUPLICATE_APPOINTMENT =
+            "Another appointment with the same date and time already exists in the app";
```
Comment (dvdweien) [inline src/main/java/seedu/address/logic/commands/EditAppointmentCommand.java:43]: thanks man

File: src/main/java/seedu/address/logic/commands/EditPropertyCommand.java [hunk 2]
```diff
@@ -71,7 +71,8 @@
             + PREFIX_TAGS + "Urgent, 4 bedrooms";
 
     public static final String MESSAGE_SUCCESS = "Edited property: %1$s";
-    public static final String MESSAGE_DUPLICATE_PROPERTY = "This property already exists in the app";
+    public static final String MESSAGE_DUPLICATE_PROPERTY =
+            "Another property with the same address and postal code already exists in the app";
```
Comment (dvdweien) [inline src/main/java/seedu/address/logic/commands/EditPropertyCommand.java:75]: thanks for this too!

**Review by dvdweien — COMMENTED** [review]
Just a quick question, do the photos have a transparent background?

**Comment by w2vgd** [comment]
> Just a quick question, do the photos have a transparent background?

No, but because our UG has a white background, it shouldn't matter?

**Review by dvdweien — APPROVED** [review]
LGTM

**Pull request merged**